content/components/dialog.mdx

```diff
@@ -13,8 +13,6 @@ export default ComponentLayout
 import {Box} from '@primer/react'
 import {Caption} from '@primer/gatsby-theme-doctocat'
 
-## Overview
```

Comment:
removed bc this is also what the overarching tab is called, so felt confusing. Usage is a section underneath, so can't dupe that here either